### PR TITLE
Updated stat_traker class to pass spec harness

### PR DESCRIPTION
**Pull request by EricLarson2020**
Updated stat_traker class to pass spec harness

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -66,8 +66,8 @@ def highest_scoring_visitor
   end
 
 
-  def best_offense(id)
-    @league_repository.best_offense(id)
+  def best_offense
```
Comment (janegreene) [inline lib/stat_tracker.rb:69]: Good catch!

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -96,19 +96,19 @@ def fewest_goals_scored(id)
   end
 
   def percentage_visitor_wins
-    @team_repository.percentage_visitor_wins
+    @game_repository.percentage_visitor_wins
```
Comment (JBelta) [inline lib/stat_tracker.rb:99]: Thanks for correction!

File: lib/stat_tracker.rb [hunk 3]
```diff
@@ -117,8 +117,8 @@ def average_goals_by_season
   end
 
 
-  def worst_offense(id)
-    @league_repository.worst_offense(id)
+  def worst_offense
```
Comment (JBelta) [inline lib/stat_tracker.rb:120]: Stick with no arguments.

**Review by janegreene — APPROVED** [review]
Updates fixed missing end and additional arguments, now passing more spec harness tests!

**Pull request merged**